extensions/mailer/runtime/src/main/java/io/quarkus/mailer/runtime/MailConfig.java

```diff
@@ -138,4 +148,43 @@
      */
     @ConfigItem(defaultValue = "false")
     public boolean multiPartOnly;
+
+    /**
+     * Enables DKIM (DomainKeys Identified Mail)
+     */
+    @ConfigItem
+    public Optional<DKIMSignConfig> dkim;
```

Comment:
I had all sorts of issues with optional config groups but maybe you will be luckier :).

Reply:
Yeah... I was just trying, does not seem to be what I need. This PR is still in draft, I need to add tests and doc.